app/components/ui/molecules/DashboardListItem.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { Flex, Box } from 'grid-styled'
+import { differenceInCalendarDays, format } from 'date-fns'
+import { th } from '@pubsweet/ui-toolkit'
+import PropTypes from 'prop-types'
+
+const dashboardStatusColor = (status, theme) =>
+  ({
+    1: theme.colorTextSecondary,
+    2: theme.colorError,
+    3: theme.colorSuccess,
+  }[status])
+
+const dashboardStatusText = status =>
+  ({
+    1: 'Waiting for decision',
+    2: 'Rejected',
+    3: 'Approved',
+  }[status])
+
+const dashboardDateText = date => {
+  const diffDays = differenceInCalendarDays(new Date(), date)
+  if (diffDays < 0) {
+    return 'Invalid date'
```

Comment:
Would it make more sense to say "Today" for cases like this which could occur if the user's computer time is slightly out?

Reply:
As far as I know, `new Date()` should be in UTC and as long as we store the date in UTC as well this should not happen unless there is a problem somewhere else in the app. I was thinking that if we show `Invalid date` to the user he might report it so we know there is a problem. If we show `Today` we might never know there is a problem, but I understand that it's nicer to show this to the user.

Reply:
I think Tam was considering more of a problem with clock drift  (see http://www.faqs.org/rfcs/rfc958.html )  rather than timezones.